DemoAppSwiftUI/SceneDelegate.swift

```diff
@@ -4,4 +4,11 @@
 
 import UIKit
 
-class SceneDelegate: NSObject, UIWindowSceneDelegate {}
+class SceneDelegate: NSObject, UIWindowSceneDelegate {
+    
+    func sceneWillResignActive(_ scene: UIScene) {
```

Comment:
Wouldn't have known where to reset this, but this is an interesting place. Cool!

Reply:
There's an alternative to do it in the App struct events, but it didn't work reliably.